.jshintrc

```diff
@@ -31,7 +31,7 @@
   "undef"         : true,     // Require all non-global variables be declared before they are used.
   "unused"        : true,     // Warn when variables are created but not used.
   "trailing"      : true,     // Prohibit trailing whitespaces.
-  "es3"           : true,     // Prohibit trailing commas for old IE 
+  "es3"           : false,     // Prohibit trailing commas for old IE 
```

Comment:
the comment can stay as it is; those are stock comments from default jshintrc and they all say what happens if the param is true (there are others that are false that aren't listed here)
I'm not psyched about relaxing this restriction either, but I figure it's a mechanical change to revert back later, if necessary (just flip this switch and see which jshint tests fail), and I was wasting a lot of cycles fixing dangling commas when changing around tests and other things.